packages/calcite-components/.stylelintrc.cjs

```diff
@@ -1,13 +1,7 @@
 // @ts-check
 
 // ⚠️ AUTO-GENERATED CODE - DO NOT EDIT
-const customFunctions = [
-  "get-trailing-text-input-padding",
-  "medium-modular-scale",
-  "modular-scale",
-  "scale-duration",
-  "small-modular-scale"
-];
+const customFunctions = [];
```

Comment:
@jcfranco this keeps happening for some reason 🤔

Reply:
Yeah, I've noticed it in a few PRs, but haven't been able to reproduce locally. 😞